front/src/apollo.tsx

```diff
@@ -65,7 +66,27 @@ const errorLink = onError(({ graphQLErrors, operation, forward }) => {
 
 export const apiClient = new ApolloClient({
   link: from([errorLink, withAuthHeadersLink, apiLink]),
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
+    typePolicies: {
+      CommentThread: {
```

Comment:
what does it do? + looks like we are writing comment specific logic in an abstract layer, should we at least extract this config to the comment module and import it here? or better can we activate this from the comment module (doesn't look to be easily doable)